tests/test_tlsfuzzer_helpers.py

```diff
@@ -242,3 +242,15 @@ def test__init__(self):
         var = AutoEmptyExtension()
 
         self.assertIsInstance(var, AutoEmptyExtension)
+
+
+class TestProtocolNameToTuple(unittest.TestCase):
+    def test_tls(self):
+        self.assertEqual((3, 1), protocol_name_to_tuple("TLSv1.0"))
```

Comment:
the interface looks like a good source of errors due to typos (e.g., it is easy to add a new protocol with a copy-paste error and no notice). A more elaborate testing here seems useful.

Reply:
I'm not sure if I understand: if the name is unknown then the `protocol_name_to_tuple` will raise an exception, while adding entries to `protocol_name_to_tuple` will require careful checking if the names are correct irrespective if translation is implemented as a lookup table or as a wall of if's

Reply:
My point is that the additional tests could help prevent a copy-paste error such as:
"tlsv1.3": (3, 4),
"tls1.3": (3, 4),
"tlsv1.4": (3, 4),
"tls1.4": (3, 5)}

Reply:
on one hand it's just as likely to make such copy-past errors in the test cases
on the other hand, double booking is used specifically to make it less likely...

I'll add them